src/test/ui/parser/let-binop.stderr

```diff
@@ -0,0 +1,39 @@
+error[E0067]: can't reassign to a uninitialized variable
+  --> $DIR/let-binop.rs:2:15
+   |
+LL |     let a: i8 *= 1;
+   |               ^^ help: replace with `=` to initialize the variable
+
+error[E0067]: can't reassign to a uninitialized variable
+  --> $DIR/let-binop.rs:4:11
+   |
+LL |     let b += 1;
+   |           ^^
+   |
+help: replace with `=` to initialize the variable
+   |
+LL |     let b = 1;
+   |           ^
+help: remove to reassign to a previously initialized variable
+   |
+LL |      b += 1;
+   |    --
+
+error[E0067]: can't reassign to a uninitialized variable
```

Comment:
I would prefer different wording here, but I'm not sure what :-/

Reply:
I agree the wording isn't ideal but I don't have any better ideas right now. The E0067 error description is: `An invalid left-hand side expression was used on an assignment operation.` however I think it's harder to understand. I might be biased though :)